requirements.txt

```diff
@@ -15,12 +15,12 @@ pyyaml==5.4.1
 six==1.15.0
 texttable==1.6.3
 tornado==6.1
-asyncpg==0.21.0
+asyncpg==0.22.0
 jinja2==2.11.3
 psutil==5.8.0
 pydantic==1.7.3
 typing_inspect==0.6.0
-importlib_metadata==2.1.0
+importlib_metadata==3.4.0
```

Comment:
on `inmanta-core` this one is still at 2.1.0?

Reply:
I think this is recent bump then

Reply:
No. Version 2.1.0 was released on the 24th of november.